PhpUnit/DefinitionHasMethodCallConstraint.php

```diff
@@ -27,13 +33,17 @@ public function evaluate($other, $description = '', $returnResult = false)
             );
         }
 
-        foreach ($other->getMethodCalls() as $methodCall) {
-            list($method, $arguments) = $methodCall;
+        for ($i = 0; $i < $iMax = count($methodCalls = $other->getMethodCalls()); $i++) {
```

Comment:
This is more related to the code style and practice - but since it is your library, I will not argue about that :)